grails-forge-core/src/main/java/org/grails/forge/feature/view/json/ViewJson.java

```diff
@@ -75,23 +75,26 @@ public void apply(GeneratorContext generatorContext) {
         generatorContext.addBuildscriptDependency(Dependency.builder()
                 .groupId("org.grails.plugins")
                 .lookupArtifactId("views-gradle")
-                .buildscript());
+                .buildSrc());
 
-        generatorContext.addBuildPlugin(GradlePlugin.builder().id("org.grails.plugins.views-json").build());
+        generatorContext.addBuildPlugin(GradlePlugin.builder()
+                .id("org.grails.plugins.views-json")
+                .useApplyPlugin(true)
+                .build());
 
         generatorContext.addDependency(Dependency.builder()
                 .groupId("org.grails.plugins")
                 .artifactId("views-json")
-                .compile());
+                .implementation());
         generatorContext.addDependency(Dependency.builder()
                 .groupId("org.grails.plugins")
                 .artifactId("views-json-templates")
-                .compile());
+                .implementation());
```

Comment:
I'm not sure the `views-json-templates` dependency should be included by default with the `views-json` feature.
I think the templates in `view-json-templates` might only be relevant when using MongoDB.
But that can be the topic for another PR.

Reply:
Sounds good.

Reply:
https://github.com/grails/grails-forge/issues/443